src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -753,7 +753,10 @@ private function registerDebugConfiguration(array $config, ContainerBuilder $con
         $container->getDefinition('debug.debug_handlers_listener')->setPrivate(true);
 
         if (class_exists(Stopwatch::class)) {
-            $container->register('debug.stopwatch', Stopwatch::class)->addArgument(true)->setPrivate(true);
+            $container->register('debug.stopwatch', Stopwatch::class)
+                ->addArgument(true)
+                ->setPrivate(true)
+                ->addTag('kernel.reset', array('method' => 'reset'));
```

Comment:
the method exists since 3.4, so I think we should raise the dep in composer.json

Reply:
Done.